loadtest/control/actions.go

```diff
@@ -471,6 +471,40 @@ func SearchUsers(u user.User) UserActionResponse {
 	})
 }
 
+// AutoCompleteUsers returns "user found" error when the auto completion is
+// done successfully. In that case UserActionResponse.Info carries the user
+// name information.
+func AutoCompleteUsers(u user.User) UserActionResponse {
+	channel, err := u.Store().CurrentChannel()
+	if err != nil {
+		return UserActionResponse{Err: err}
+	}
+
+	user, err := u.Store().RandomUser()
+	if err != nil {
+		return UserActionResponse{Err: err}
+	}
+
+	cutoff := 2 + rand.Intn(len(user.Username)/2)
```

Comment:
Wondering if there is any data behind this ? Otherwise, might be good to write a comment on it.

Reply:
Actually I didn't use any data for this one. It's just an estimation that expects at some point the users may find what they are searching for.

Reply:
Yes, we'll review all these details later on since we don't have any specific insight at the moment other than what we experience ourselves. On a real server usernames are pretty distinct so usually after a few characters we should be able to tab complete or select and add the mention.